core/database/schema/query_test.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 type querySuite struct {
-	databasetesting.ControllerSuite
+	databasetesting.DqliteSuite
```

Comment:
Thank you I didn't realise we were using sqlite for the tests. Apologies for assuming what I had written was good to go.